src/components/navbar/navbar.component.css

```diff
@@ -0,0 +1,3 @@
+.headerGlobalActionSlot > div{
```

Comment:
Prettify.

Space before the curly brace, 2 spaces indentations.

Reply:
Will take care of that.